src/App.tsx

```diff
@@ -14,11 +14,100 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+enum SortBy {
+  default,
+  name,
+  length,
+}
+
+export const App: React.FC = () => {
+  let goods = [...goodsFromServer];
+
+  const [isVisible, setStart] = useState(false);
+  const [isReverse, setReverse] = useState(false);
```

Comment:
although I also didn't follow the naming convention, apparently the convention says you should name it as`setIsVisible` and `setIsReverse`